warehouse/slave.go

```diff
@@ -405,14 +405,17 @@ func processStagingFile(job PayloadT) (loadFileIDs []int64, err error) {
 	}
 
 	for tableName, outputFile := range jobRun.outputFileWritersMap {
```

Comment:
Should we rename outputFile to loadFile? That sounds like a better name.

Reply:
Btw, should we upload them in parallel with a max number of parallel workers? Should be a small change but will improve load file generation times.

CC: @psrikanth88